org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -311,21 +315,22 @@ public void clearQueue(String qName) throws EventException {
 			}
 		}
 	}
-	
+
+	@SuppressWarnings("squid:S3776")
 	@Override
 	public boolean reorder(U bean, String queueName, int amount) throws EventException {
-		
+
 		if (amount==0) return false; // Nothing to reorder, no exception required, order unchanged.
-		
+
 		PauseBean pbean = new PauseBean(queueName);
 		pbean.setMessage("Pause to reorder '"+bean.getName()+"' "+amount);
-		
+
 		IPublisher<PauseBean> publisher = createPausePublisher();
 		final boolean isAlreadyPaused = isQueuePaused(queueName);
 		if (!isAlreadyPaused) publisher.broadcast(pbean);
-		
+
 		try {
-			
+
 			// We are paused, read the queue
 			List<U> submitted = getQueue(queueName, null);
 			if (submitted==null || submitted.size()<1) throw new EventException("There is nothing submitted waiting to be run\n\nPerhaps the job started to run.");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1155)